### PR TITLE
LPS-171870| Test Fix | Master

### PR DESCRIPTION
**Pull request by pedroamarall**
Ticket:
https://issues.liferay.com/browse/LPS-171870?filter=-1

### CODE REVIEW

**Comment by pedroamarall** [comment]
ci:test:sf

**Comment by pedroamarall** [comment]
ci:test:sf

File: modules/apps/calendar/calendar-web-test/src/testFunctional/paths/ManageCalendar.path [hunk 1]
```diff
@@ -12,7 +12,7 @@
 <tbody>
 <tr>
 	<td>COLOR_BY_COLUMN</td>
-	<td>//tr[${key_rowNumber}]//td[contains(@class, 'lfr-color-column')]//*[contains(@style,'${key_color}')]</td>
+	<td>//td[contains(@class, 'lfr-color-column')]//*[contains(@style,'${key_color}')]</td>
```
Comment (epark18) [inline modules/apps/calendar/calendar-web-test/src/testFunctional/paths/ManageCalendar.path:15]: When you're changing a path, always consider the macros and tests that can be affected. Removing the `key_rowNumber` variable might lead to other failures. 
Also, if you make this type of change; this path will be located if any calendar shares the same color. So for example, if you have two calendars that use the same color on the page, it'll always default to the first one.

**Review by epark18 — COMMENTED** [review]
@pedroamarall Hey, remember to attach your index.html on the ticket. When you change macros or paths you should also attach index.html(test results) of tests that use those macros/paths to ensure other tests don't break to fix one. 
I left a comment for the path change.
Lastly your commit messages have some typos and can be a little more specific. Consider:
LPS-171870 Change paths to locate calendar colors
LPS-171870 Update test to reflect path changes

**Comment by pedroamarall** [comment]
ci:test:sf

File: modules/apps/calendar/calendar-web-test/src/testFunctional/paths/ManageCalendar.path (hunk 1 above)
Comment (pedroamarall) [reply modules/apps/calendar/calendar-web-test/src/testFunctional/paths/ManageCalendar.path]: Hello @epark18, I appreciated your feedback, I changed now.

**Comment by pedroamarall** [comment]
ci:test:sf

**Comment by epark18** [comment]
@pedroamarall Great job on this. 
One thing to mention is that it looks like you pulled from upstream and included all those changes onto a commit or you added a lot of untracked files. Either case, we don't want to add code we did not intend from our own changes. Could you fix this commit so it only includes your change? Thank you!
https://github.com/liferay-bpm-qa/liferay-portal/pull/442/commits/aad888d61029faedffb464a2564fd72f884887dc

**Comment by pedroamarall** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**